src/test/java/com/casper/sdk/service/CasperServiceTests.java

```diff
@@ -718,6 +728,226 @@ void queryBalanceDetails() throws IOException, DynamicInstanceException {
         assertThat(result.getHolds().get(0).getProof(), is("0100000006328c317bc7f9fd7d2b5fa9cf3b4c09fc5a8f"));
     }
 
+    @Test
+    void stateGetEntityAccount() throws NoSuchAlgorithmException {
+
+        mockNode.withRcpResponseDispatcher()
+                .withMethod("state_get_entity")
+                .withBody("$.params.entity_identifier.PublicKey", "0138329930033bca4773a6623574ad7870ee39c554f153f15609e200e50049a7de")
+                .thenDispatch(getClass().getResource("/entity/getstateentity-account-result.json"));
+
+        final StateEntityResult stateEntityResult = casperServiceMock.getStateEntity(
+                new PublicKeyEntityIdentifier(PublicKey.fromTaggedHexString("0138329930033bca4773a6623574ad7870ee39c554f153f15609e200e50049a7de"))
+        );
+
+        assertThat(stateEntityResult.getApiVersion(), is("2.0.0"));
+
+        assertInstanceOf(AddressableEntity.class, stateEntityResult.getEntity());
+
+        AddressableEntity entity = (AddressableEntity)stateEntityResult.getEntity();
+
+        assertInstanceOf(Account.class, entity.getEntity().getEntityAddressKind());
+
+        assertThat(entity.getEntity().getByteCodeHash(), is("byte-code-0000000000000000000000000000000000000000000000000000000000000000"));
+        assertThat(entity.getEntity().getPackageHash(), is("package-22138fb22c624016fd73cd2abd3285ccd27cb80e20c29f3d94d34e95ee182030"));
+
+        Account account = (Account) entity.getEntity().getEntityAddressKind();
+        assertThat(account.getAccount(), is("account-hash-aab0da01340446cee477f28410f8af5d6e0f3a88fb26c0cafb8d1625f5cc9c10"));
+
+        assertThat(entity.getEntity().getMainPurse().getJsonURef(), is("uref-3dfdbde34845bd2e731cf39fba450745eba645b75d5feb3dcf953fd06d69bf14-007"));
+        assertThat(entity.getEntity().getAssociatedKeys().get(0).getAccountHash(), is("account-hash-aab0da01340446cee477f28410f8af5d6e0f3a88fb26c0cafb8d1625f5cc9c10"));
+        assertThat(entity.getEntity().getAssociatedKeys().get(0).getWeight(), is(1));
+        assertThat(entity.getEntity().getActionThresholds().getDeployment(), is(1));
+        assertThat(entity.getEntity().getActionThresholds().getKeyManagement(), is(1));
+        assertThat(entity.getEntity().getActionThresholds().getUpgradeManagement(), is(1));
+
+    }
+    @Test
```

Comment:
Ditto newlines between test methods. Rather see no whitespace befoe closing } and whitespace after as per Java programming guidelines